communication/src/main/java/org/jboss/da/communication/aprox/impl/AproxConnectorImpl.java

```diff
@@ -0,0 +1,76 @@
+package org.jboss.da.communication.aprox.impl;
+
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.net.URL;
+import java.net.URLConnection;
+import java.util.List;
+
+import javax.ejb.Stateless;
+import javax.xml.bind.JAXBContext;
+import javax.xml.bind.JAXBException;
+import javax.xml.bind.Unmarshaller;
+
+import org.jboss.da.communication.aprox.api.AproxConnector;
+import org.jboss.da.communication.aprox.model.GA;
+import org.jboss.da.communication.aprox.model.GAV;
+import org.jboss.da.communication.aprox.model.GAVDependencyTree;
+import org.jboss.da.communication.aprox.model.VersionResponse;
+
+@Stateless
+public class AproxConnectorImpl implements AproxConnector {
+
```

Comment:
Is the AproxConnector interface finalised or will there be other API interface support ?